ui/core/components/stat_weights_action.ts

```diff
@@ -122,6 +122,31 @@ class EpWeightsMenu extends BaseModal {
 								</a>
 							</th>
 						</tr>
+						<tr>
+							<th>Ratio</th>
+							<td class="damage-metrics type-ratio type-weight">
+							</td>
+							<td class="damage-metrics type-ratio type-ep">
```

Comment:
add to _stat_weights_action.scss 
```.results-ep-table-container .type-ratio {
  text-align: right;
}```

To make these boxes align with the text columns under them